src/conf.ml

```diff
@@ -3,7 +3,10 @@ let staging_repo = "ocurrent/opam-staging"
 
 let public_repo = "ocaml/opam"
 
-let password_path = "/run/secrets/ocurrent-hub"
+let password_path =
+  let open Fpath in
+  let root = v (if Sys.win32 then "C:\\ProgramData\\Docker" else "/run") in
```

Comment:
Why's this in `Docker`? I'd take the root as `"/run/secrets"` on Unix and `Option.value (Sys.getenv_opt "ProgramData") ~default:"C:\ProgramData")` on Windows and then put the secrets in `ocurrent-hub` under that.

Reply:
@dra27 we haven't really resolved that conversation, are you convinced by my explanation?